reference/unity_to_godot.rst

```diff
@@ -0,0 +1,177 @@
+|                   |                                                                                   | | Animation tool                                                                                               |
+|                   |                                                                                   | | Shader graphs                                                                                                |
+|                   |                                                                                   | | GDScript                                                                                                     |
+|                   |                                                                                   | | Integrated debugger                                                                                          |
++-------------------+-----------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
+
+
+The editor
+----------
+
+Godot Engine provides a rich-featured editor that allows you to build your games. The pictures below display both editors with colored blocks to indicate common functionalities.
+
+.. image:: /img/unity-gui-overlay.png
+.. image:: /img/godot-gui-overlay.png
+
+
+Note that Godot editor allows you to dock each panel at the side of the scene editor you wish.
+
+As you can see, there are some similarities between both editors: first of course, the scene which allows you to display and organize your game elements on screen. Second, the inspector which allows you to edit the parameters of the selected object. Also, the Hierarchy panel in Unity finds its correspondance with Godot's Scene tree panel, although we will see in the next section the differences between how Unity and Godot manage scenes and scene elements.
+
+Unity and Godot both directly use the folder architecture you define for your project. There is no real good solution, and every folder architecture will work for both engines. You can thus continue with your current way to organize your project. However, we have some advices to share a bit further, in order to match your project architecture and Godot scene system.
+
+To finish, the toolbar at the top of the screen is also similar between Unity and Godot. However, Godot provides some new buttons aside the classic "Play", "Pause" and "Stop" buttons. These buttons are introduced in the `Toolbar <introduction/editor#toolbar>`_ section as they a bit out of the scope of this page.
+
+The scene system
+----------------
+
+This is the most important difference between Unity and Godot, and actually the favourite feature of most Godot users.
+
+Unity's scene system generally consist in embedding all the required assets in a scene, and link them together by setting components and scripts to them. 
```

Comment:
I would try to remove extra words such as "generally" as much as possible, and text short and concise